app/uk/gov/hmrc/exports/repositories/SubmissionRepository.scala

```diff
@@ -93,11 +92,9 @@ class SubmissionRepository @Inject()(implicit mc: ReactiveMongoComponent, ec: Ex
 
   private def performUpdate(query: JsObject, update: JsObject): Future[Option[Submission]] =
     findAndUpdate(query, update, fetchNewObject = true).map { updateResult =>
-      if (updateResult.value.isEmpty) logDatabaseUpdateError(updateResult)
+      if (updateResult.value.isEmpty) {
+        updateResult.lastError.foreach(_.err.foreach(errorMsg => logger.error(s"Problem during database update: $errorMsg")))
+      }
       updateResult.result[Submission]
     }
-
-  private def logDatabaseUpdateError(res: FindAndModifyResult): Unit =
```

Comment:
FindAndModifyResult is depracated and in next versions will be unavailable to use. There was some problem with import. This method has been used in just one place so inline solved the problem.